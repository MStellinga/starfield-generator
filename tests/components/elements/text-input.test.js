/* eslint-disable jest/valid-expect */
import { h } from 'preact';
import TextInput from '../../../src/components/elements/text-input';
import render from 'preact-render-to-string';

describe('Test input field', () => {
	test('Text input renders correctly', () => {
        const tree = render(<TextInput value="123" />);
        expect(tree).toMatchSnapshot();	
    });
});
