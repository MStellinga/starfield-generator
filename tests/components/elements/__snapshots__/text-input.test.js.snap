// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test input field Text input renders correctly 1`] = `"<input type=\\"text\\" pattern=\\"-?[0-9\\\\.]{1,5}\\" class=\\"smallfield\\" value=\\"123\\" />"`;
